pandas/_libs/window/indexers.pyx

```diff
@@ -38,17 +39,20 @@ def calculate_variable_window_bounds(
     closed : str
         string of side of the window that should be closed
 
+    step : int64
+        Spacing between windows
+
     index : ndarray[int64]
         time series index to roll over
 
     Returns
     -------
-    (ndarray[int64], ndarray[int64])
+    (ndarray[int64], ndarray[int64], ndarray[int64])
```

Comment:
can you document what these arrays are here

Reply:
My understanding is that @mroeschke would like to punt on group-by (and other) behaviors, which would avoid the need for this 3rd tuple item for the time being. If this is the agreed way to go, I'll go back to the original API here.

Reply:
Yeah I think it would be okay to just focus on rolling with an integer step size to avoid the 3rd tuple item.

Reply:
This `Return`s doc change can be reverted now since this just returns start, end